Documentation landing page before login (#36)
landinfotech/mycivitas: docs/src/user/manual/landing-page.md

### Key Features

- **All-Inclusive Platform:** MyCivitas provides a one-stop solution for users with all community asset management needs. From property details to infrastructure, easily record and manage diverse assets on a single platform.

- **User-Friendly Interface:** MyCivitas's intuitive interface ensures that users, regardless of technical expertise, can navigate and utilize MyCivitas effortlessly. The platform is designed for accessibility and ease of use.

- **Affordability:** We understand the unique needs of small communities. MyCivitas offers affordable subscription plans, making robust asset management accessible to organizations of all sizes.

- **Powerful Information System:** MyCivitas is built on a powerful information system that allows the user to organize, search, and analyze the user's assets efficiently. Gain valuable insights to make informed decisions.

![landing page](./img/landing-page-1.png)

### Navigation

Use the navigation bar to explore various sections of the landing page. From understanding how MyCivitas works to exploring pricing and our story, everything is at the user's fingertips.

> Easily sign in if the user is an existing user or sign up to join the MyCivitas community. MyCivitas's user-friendly registration process ensures a quick and hassle-free onboarding experience.

Reviewer: ```suggestion
Easily sign in if the user is an existing user or sign up to join the MyCivitas community. MyCivitas' user-friendly registration process ensures a quick and hassle-free onboarding experience.
```

1. **MyCivitas:** Click on `MyCivitas` to swiftly return to the top of the landing page.

2. **How Does MyCivitas Work?:** Explore the functionalities and features of MyCivitas by clicking on this option.

>     - **How Does MyCivitas Work?:** MyCivitas revolves around a centralized map interface that serves as the core of the user's community asset management. This interactive map provides a visual representation of the community's assets and infrastructure, offering a user-friendly and intuitive way to navigate and interact with the information. Keep scrolling to delve deeper into the functionalities and features that MyCivitas offers for efficient asset management.

Reviewer: ```suggestion
    - **How Does MyCivitas Work?:** MyCivitas revolves around a centralised map interface that serves as the core of the user's community asset management. This interactive map provides a visual representation of the community's assets and infrastructure, offering a user-friendly and intuitive way to navigate and interact with the information. Keep scrolling to delve deeper into the functionalities and features that MyCivitas offers for efficient asset management.
```

    ![how does mycivitas work](./img/landing-page-2.png)

>     - **Capture Your Infrastructure:** MyCivitas empowers users to build a comprehensive asset inventory with ease. Utilizing powerful mapping software, MyCivitas's data capture system is integrated with QGIS, a robust and free open-source desktop application. Leverage this combination to seamlessly capture and record detailed information about the community's infrastructure, ensuring accuracy and efficiency in asset management.

Reviewer: ```suggestion
    - **Capture Your Infrastructure:** MyCivitas empowers users to build a comprehensive asset inventory with ease. Utilising powerful mapping software, MyCivitas' data capture system is integrated with QGIS, a robust and free open-source desktop application. Leverage this combination to seamlessly capture and record detailed information about the community's infrastructure, ensuring accuracy and efficiency in asset management.
```

    ![capture your infrastructure](./img/landing-page-7.png)

>     - **Prioritize Your Assets:** MyCivitas goes beyond simple asset management by incorporating advanced functionalities. MyCivitas's system conducts thorough risk assessment analysis and preventative maintenance planning on user's data. This means the user can prioritize assets based on their criticality, allowing for strategic decision-making and proactive planning to ensure the longevity and optimal performance of their community's infrastructure.

Reviewer: ```suggestion
    - **Prioritize Your Assets:** MyCivitas goes beyond simple asset management by incorporating advanced functionalities. MyCivitas' system conducts thorough risk assessment analysis and preventative maintenance planning on user's data. This means the user can prioritise assets based on their criticality, allowing for strategic decision-making and proactive planning to ensure the longevity and optimal performance of their community's infrastructure.
```

    ![prioritize your assets](./img/landing-page-8.png)

>     - **Support Your Capital Planning:** MyCivitas equips the user with the tools to make well-informed decisions that support capital planning initiatives. By providing comprehensive data and insights, MyCivitas's platform empowers the user to strategically allocate resources, prioritize projects, and optimize their capital investments. Make informed decisions backed by accurate and up-to-date information for effective long-term planning and development.

Reviewer: ```suggestion
    - **Support Your Capital Planning:** MyCivitas equips the user with the tools to make well-informed decisions that support capital planning initiatives. By providing comprehensive data and insights, MyCivitas' platform empowers the user to strategically allocate resources, prioritise projects, and optimise their capital investments. Make informed decisions backed by accurate and up-to-date information for effective long-term planning and development.
```

    ![support your capital planning](./img/landing-page-9.png)

>     - **Set Up Work Order:** MyCivitas streamlines maintenance planning by enabling the user to set up work orders effortlessly. Whether it is ad hoc tasks or scheduled maintenance, the platform provides a user-friendly interface for planning and organizing maintenance activities. Take advantage of this feature to ensure timely and efficient management of tasks, contributing to the overall maintenance and well-being of their community's assets.

Reviewer: ```suggestion
    - **Set Up Work Order:** MyCivitas streamlines maintenance planning by enabling the user to set up work orders effortlessly. Whether it is ad hoc tasks or scheduled maintenance, the platform provides a user-friendly interface for planning and organising maintenance activities. Take advantage of this feature to ensure timely and efficient management of tasks, contributing to the overall maintenance and well-being of their community's assets.
```

    ![we are here to help](./img/landing-page-11.png)

    - **Enjoy Asset Management** MyCivitas invites users to practice effective and sustainable asset management on our affordable platform for the benefit of the community. Embrace the convenience, insights, and efficiency that MyCivitas offers, ensuring that their assets are managed with precision and care. By leveraging our platform, you contribute to the overall well-being and success of the community's infrastructure. Enjoy the journey of asset management with MyCivitas!

    ![enjoy asset management](./img/landing-page-12.png)

3. **Pricing:** Learn about our subscription plans and pricing details to choose the best fit for your needs.

    ![pricing](./img/landing-page-3.png)

    **Plans**

>     - **Silver:** Allowing up to 3 users in the organization.

Reviewer: ```suggestion
    - **Silver:** Allowing up to 3 users in the organisation.
```

>     - **Gold:** Allowing up to 5 users in the organization.

Reviewer: ```suggestion
    - **Gold:** Allowing up to 5 users in the organisation.
```

>     - **Diamond:** Allowing up to 10 users in the organization.

Reviewer: ```suggestion
    - **Diamond:** Allowing up to 10 users in the organisation.
```
